test/source/tests/gmail.ts

```diff
@@ -307,6 +307,7 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
           await gotoGmailPage(gmailPage, `/${threadId}`); // go to encrypted convo
           t.timeout(minutes(4)); // extend ava's timeout
           await GmailPageRecipe.trimConvo(gmailPage, threadId);
+          expect(await gmailPage.isElementPresent('div.aHU')).to.equal(true); // @getConvoRootEl() target element for rendering encrypted reply or forward buttons
```

Comment:
Hmm, I wanted to comment that it should be better to check if we have `compose.htm` iframe on the page (which means secure reply buttons were loaded successfully) than checking for exact selector, but on the next lines we already check for iframe presence.

It was strange for me that this test didn't fail, I ran it locally and found that for `ci.tests.gmail` account `td.Bu` is still present in Gmail code, so reply buttons work well even without our fix.

So we already have check for secure reply buttons presence and test didn't fail just because Gmail code changes affected not all users.